doc/compiler/optimizer/DataFlowEngine.md

```diff
@@ -127,4 +127,61 @@ In general using structures as our program fragments enables us to save roughly
+which expression is transparent or can be moved across the block. If the value of this expression does not change in this block, this
+expression is considered locally transparent to the block. If the value of the expression is changed in the block, the expression is
+not locally transparent to the block. The expression cannot be moved across this block.
+
+For example, there is a computation of `x+y`. Where `x+y` is computed and where `x+y` should be computed (the optimal spot) can be many
+blocks apart. Local Transparency determines if `x+y` can flow through all these blocks to the optimal spot where it should be computed.
+If the block does not change `x` or `y`, `x+y` can be moved pass this block. If `x+y` is computed earlier, and then it is commuted again,
+and `x` and `y` have not been changed in between, the two computations can be commoned by Particial Redunancy Elimination (PRE).
+The value of `x+y` can be stored in a temporary which can be used in later blocks instead of computing `x+y` again.
+
+[TR_LocalTransparency](https://github.com/eclipse/omr/blob/f2bc4f8f6eb09f6cc8fc4ba48717de4880b970e3/compiler/optimizer/LocalAnalysis.hpp#L230)
+runs before [TR_LocalAnticipatability](https://github.com/eclipse/omr/blob/f2bc4f8f6eb09f6cc8fc4ba48717de4880b970e3/compiler/optimizer/LocalAnalysis.hpp#L286):
+- `TR_LocalTransparency::_transparencyInfo[symRefNum]`: A bit vector indicates which expressions (identified by local index) will not be
+affected by the sym ref if the sym ref is changed in the block
+- `TR_LocalTransparency::_info[blockNum]._analysisInfo`: A bit vector indicates whether or not the expression (identified by local index)
+is transparent to the block
+
+### Local Anticipatability
+
+Local Transparency does not look at whether or not an expression is evaluated in a certain block. It looks at an expression regardless
+of where it is computed. Local Anticipatablility looks at the expressions that are computed in the block. If an expression is locally
+anticipatable in a block, the expression is computed in the block and can be moved out of the block without changing the value of the
+expression. For exmaple, if `x+y` is computed in `block_10`, the expression `x+y` can be a candidate of being locally anticipatable in
+`block_10`. If `x+y` is not computed in `block_10`, the expression `x+y` will not be in the Local Anticipatablility set.
+
+[TR_LocalAnticipatability](https://github.com/eclipse/omr/blob/f2bc4f8f6eb09f6cc8fc4ba48717de4880b970e3/compiler/optimizer/LocalAnalysis.hpp#L286)
+runs after [TR_LocalTransparency](https://github.com/eclipse/omr/blob/f2bc4f8f6eb09f6cc8fc4ba48717de4880b970e3/compiler/optimizer/LocalAnalysis.hpp#L230)
+- `TR_LocalAnticipatability::_info[blockNum]._analysisInfo`: A bit vector indicates whether or not the expression (identified by local
+index) that is being computed in the block is locally anticipatable
```

Comment:
You may want to make it clear that the `_info` represents the upward anticipatable notion, meaning it is "would be changed if it were computed at the very start of the block" (to use terminology similar to what you used in the "downward exposed" case)